packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useFetchMappingExtractionStatus.ts

```diff
@@ -66,12 +62,12 @@ export const useFetchMappingExtractionStatus = ({
 }: MappingExtractionStatusProps) => {
   const extractionClient = useExtractionClient();
   const { mappingIdJobInfo } = useExtractionStateJobContext();
-  const jobId = mappingIdJobInfo.get(mapping.id);
+  const jobId = mappingIdJobInfo.get(mapping.id) ?? "";
```

Comment:
`jobId` should never be an empty string. It should either be null / undefined or a valid value (guid).

Reply:
Made it more clear in the code.